packages/suite/src/components/suite/layouts/WelcomeLayout/WelcomeLayout.tsx

```diff
@@ -193,7 +130,7 @@ export const WelcomeLayout = ({ children }: WelcomeLayoutProps) => {
 
                     <Row height="100%" width="100%" data-testid="@welcome-layout/body">
                         <ElevationDown>
-                            <Left />
+                            <LoggedOutSidebar />
```

Comment:
Since there are multiple layouts for the state when "you are not logged in". I need to include this component here also. Above, it is rendered also in the logged out layout.